portlets/src/main/webapp/vue-app/badgesOverview/components/BadgesOverviewDrawer.vue

```diff
@@ -21,7 +21,17 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     body-classes="hide-scroll decrease-z-index-more"
     right>
     <template slot="title">
-      {{ $t('exoplatform.gamification.badgesByDomain') }}: {{ badge && badge.domainLabel }}
+      <div class="d-flex">
+        {{ $t('exoplatform.gamification.badgesByDomain') }}:
+        <v-tooltip bottom>
+          <template #activator="{ on }">
+            <div v-on="on" class="ms-1">
+              {{ domainLabelToDisplay }}
```

Comment:
Can you use `text-truncate` please

Reply:
I tried to use text-truncate on the program name only and it didn't work (the header contains badges by domains + program name)